res/values/strings.xml

```diff
@@ -461,10 +461,10 @@
     <string name="pref_auto_folder_moves">Automatic moves to DeltaChat folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
     <string name="pref_email_interaction_title">Email interaction</string>
-    <string name="pref_show_emails">Show classic emails</string>
-    <string name="pref_show_emails_no">No, chats only</string>
+    <string name="pref_show_emails">Show all emails</string>
+    <string name="pref_show_emails_no">No, only chats via Delta Chat</string>
```

Comment:
I think this line is too long to be displayed nicely. How about

"No, only Delta Chats"

Reply:
I did not try this, but I do not think that non-techie people would understand the term "Delta Chats" here.

Reply:
Of course your idea is way better than the original because that's a smart way to use the term "Delta Chat" without using "Chat" twice :+1:.